.gitignore

```diff
@@ -230,6 +230,11 @@ CTAGS
 tags
 TAGS
 
+# Files generated by GNU Global (gtags)
```

Comment:
Is this intended?  If it is, it should probably be in a separate PR.

Reply:
OK I'll send a separate PR. Will update my commit soon.